Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -198,4 +202,25 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
         navigationItem.rightBarButtonItem?.isEnabled = titleText != nil
     }
     
+    // MARK: UIImagePickerControllerDelegate
+    
+    func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [String : Any]) {
+        guard let image = info[UIImagePickerControllerOriginalImage] as? UIImage else { return }
+        
+        let username = client.sessionManager.focusedUserSession?.username
+        guard let uploadController = ImageUploadTableViewController.create(image, username: username, delegate: self) else { return }
+        
+        picker.pushViewController(uploadController, animated: true)
+    }
+    
+    func imagePickerControllerDidCancel(_ picker: UIImagePickerController) {
+        dismiss(animated: true, completion: nil)
+    }
+    
+    // MARK: ImageUploadDelegate
+    
+    func imageUploaded(link: String, altText: String) {
+        bodyField.replace(left: "![\(altText)](\(link))", right: nil, atLineStart: true)
+    }
```

Comment:
Any change we can put this in something that can be shared elsewhere? Would be awesome to have this in the commenting eventually too!